NEWS.md

```diff
@@ -1,197 +1,227 @@
+## 30.0.0 2020-05-15
```

Comment:
Can you add a link to the upgrading notes?

Reply:
Like this?
```
See [upgrade notes](doc/upgrading.md)
```

Reply:
Also saw this:
```
dam@fry:~/folio/raml-module-builder$ pandoc NEWS.md -o NEWS.html
[WARNING] Could not convert TeX math '{myuniversity}_', rendering as TeX:
  {myuniversity}_
                 ^
  unexpected eof

```

So I'd like to backquote those curly braces

Reply:
`See [upgrade notes](doc/upgrading.md#version-300)`

Reply:
yes. What I just did. Releasing now